src/components/Editor/TableOfContents.vue

```diff
@@ -69,6 +69,9 @@ export default {
 	padding: 0 10px;
 	color: var(--color-main-text-maxcontrast);
 	--animation-duration: 0.8s;
+	max-height: 73vh;
+	overflow-x: hidden;
```

Comment:
Mh, does this work with long headings? What I meant is that the scrollbar should not be there if it is not needed for viewing the whole heading, but for sure it should be there once the heading is too long for being displayed completely within the container without scrolling.